packages/addon-dev/sample-typescript-babel.config.cjs

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const { resolve } = require;
+
+module.exports = {
+  presets: [resolve('@babel/preset-typescript')],
+  plugins: [
+    [
+      resolve('@babel/plugin-transform-typescript'),
+      {
+        allowDeclareFields: true,
+        onlyRemoveTypeImports: true,
+        // Default enums are IIFEs
+        optimizeConstEnums: true,
+      },
+    ],
+    [
+      resolve('@babel/plugin-proposal-decorators'),
```

Comment:
We don't need to specify the class-properties plugin.

By only specifying the decorators plugin, we leave properties "native" and untranspiled, since all browsers support class properties now.

Reply:
This code is fine, but your explanation for why it's fine is incorrect:

> since all browsers support class properties now.

Browser support has nothing to do with it. That is a concern for apps, not addons.

The only reason we need to transpile the decorators here is that acorn doesn't parse them, so rollup blows up. As soon as that is fixed, we should drop decorator transpilation, and that won't depend on whether any browsers have implemented decorators.